docs/source/using-discovery/usingslurm.rst

```diff
@@ -4,97 +4,634 @@
 
 .. list-table::
-   :widths: 20 100
+   :widths: 35 85
    :header-rows: 1
 
    * - Slurm Command
      - Function
    * - ``sinfo <options>``
-     - View partition and node information. Use option -a to view all partitions.
+     - View partition and node information; use option ``-a`` to view all partitions
    * - ``smap <options>``
      - View details about the cluster in a visual format
 
 .. _submitting_jobs:
-
-Submitting Jobs
+Submitting jobs
 ================
 
 There are two main commands for submitting jobs to Discovery: ``srun`` and ``sbatch``.
 To run a job interactively, use ``srun``. To submit a job to run in the background with a script, use ``sbatch``.
 
 .. list-table::
-   :widths: 20 100
+   :widths: 30 90
    :header-rows: 1
 
    * - Slurm Command
      - Function
    * - ``srun``
-     - Run an interactive job on the cluster. See :ref:`using_srun`
+     - Run an interactive job on the cluster
    * - ``sbatch <scriptname.script>``
-     - Submit a script to the scheduler for running a job. See :ref:`using_sbatch`
+     - Submit a script to the scheduler for running a job
    * - ``scancel <jobid>``
      - Cancel a pending or running job on the cluster
 
-Monitoring Jobs
+Using srun
+-----------
+Use the Slurm command ``srun`` to allocate an interactive job. This means you use specific options with ``srun`` on the command line to tell Slurm what resources are needed to run your job, such as number of nodes, amount of memory, and amount of time. Enter ``srun`` command and options on the command line, the and press ``Return``. Slurm will find, and then allocate, the specified resources. Depending on the specifications, this process may take a few minutes. All ``srun`` options can be found in the `Slurm documentation`_. The following image shows an example of an ``srun`` command as run on a command line.
+
+.. image:: /images/srun_example.jpg
+  :alt: image of the command line showing an example ``srun`` command
+
+Examples using srun
+^^^^^^^^^^^^^^^^^^^
+This section details a few examples using ``srun``. You should first review the :ref:`hardware_overview` and :ref:`partition_names` sections
+to be familiar with the available hardware and partition limits on Discovery. This way, you can tailor your request to fit both the needs of your job
+and the limits of the partitions. For example, if you specify ``--partition=short`` and ``--time=01:00:00``, you'll get an error because the
+time you've specified exceeds the limit for that partition. Also keep in mind that while these examples are all valid, general examples, they might not work
+for your particular job.
+
+This simple ``srun`` example is to move to a _compute_ node after you first log into Discovery::
+
+ srun --pty /bin/bash
+
+To request one node and one task for 30 minutes with X11 forwarding on the short partition, type::
+
+ srun --partition=short --export=ALL --nodes=1 --ntasks=1 --x11 --mem=10G --time=00:30:00 --pty /bin/bash
```

Comment:
I think we want to remove the --export=ALL from suggested use in srun but @manasvita can confirm

Reply:
Yes, I confirm that we do want to remove the --export=ALL option from the srun command @visionjo. Thanks for remembering that @SerenaCaplins